node/tests/RedisClient.test.ts

```diff
@@ -27,7 +27,7 @@ type Context = {
     client: RedisClient;
 };
 
-const TIMEOUT = 10000;
+const TIMEOUT = 50000;
```

Comment:
why?

Reply:
It is a total IT timeout, 10 sec is too low. Sometimes it runs longer and causes test failures.